data/campaigns/Winds_of_Fate/story/Morogor_Dialect.txt

```diff
@@ -78,7 +78,7 @@ May your wings catch the winds to soar.
 See "dominant".
 
 - Dominant -
-The leader of a drake flight, chosen through rigorous tests of group hunting ability followed by a contest of military leadership known as the contention.
+The leader of a drake flight, chosen through rigorous tests of group hunting ability followed by a contest of military leadership known as the Contention.
```

Comment:
Should this really be made capitalized here, if it is never capitalized in drake speech anywhere in the campaign?

Reply:
I don't really care one way or another, it just seemed weird that most of the other things in the file are capitalized when used in other entries.